custom-connectors/Tikit/apiDefinition.swagger.json

```diff
@@ -0,0 +1,421 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tikit",
+    "description": "This is a ticketing management system.",
+    "version": "1.0",
+    "contact": {
+      "name": "Celebaltech Support",
+      "url": "https://www.celebaltech.com/support",
+      "email": "support@celebaltech.com"
+    }
+  },
+  "host": "tikit.azurewebsites.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.celebaltech.com"
+    },
+    {
+      "propertyName": "Privacy Policy",
+      "propertyValue": "https://www.celebaltech.com/privacypolicy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "AI;Sales and CRM"
+    }
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/Ticket/AddTicket": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default"
+          }
+        },
+        "summary": "AddTicket",
```

Comment:
@1MrSharma All the summary and description values in the swagger are expected to be free from grammatical and spell errors. Please provide valid summary and description values for all the properties. **This applies to all the summary and description values in the swagger.**

Reply:
Updated the changes required.